src/include/storage/copier/copy_node_arrow.h

```diff
@@ -17,17 +17,24 @@ class CopyNodeArrow : public CopyStructuresArrow {
               tableID},
           nodesStatisticsAndDeletedIDs{nodesStatisticsAndDeletedIDs} {}
 
-private:
+protected:
     inline void updateTableStatistics() override {
         nodesStatisticsAndDeletedIDs->setNumTuplesForTable(tableSchema->tableID, numRows);
     }
 
     void initializeColumnsAndLists() override;
 
-    void populateColumnsAndLists() override;
+    virtual void populateColumnsAndLists() override;
```

Comment:
Remove virtual, it is already marked as override.

Reply:
Done